src/components/WalletDropdown/DefaultMenu.tsx

```diff
@@ -0,0 +1,263 @@
+  width: 288px;
+  background-color: ${({ theme }) => theme.accentAction};
+  color: white;
+  font-weight: 600;
+  font-size: 16px;
+  cursor: pointer;
+`
+
+const IconContainer = styled.div`
+  display: inline-block;
+  float: right;
+  & > a {
+    margin-right: 8px;
+  }
+  & > a:last-child {
+    margin-right: 0px;
+    ${IconHoverText}:last-child {
+      left: 0px;
+    }
+  }
+`
+
+const Divider = styled.div`
+  border: 0.5px solid ${({ theme }) => theme.backgroundOutline};
+  margin-top: 16px;
+  margin-bottom: 16px;
+`
+
+const ToggleMenuItem = styled.button`
+  background-color: transparent;
+  margin: 0;
+  border: none;
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 0px;
+  justify-content: space-between;
+  font-size: 14px;
+  font-weight: 400;
+  width: 100%;
+  margin-bottom: 8px;
+  color: ${({ theme }) => theme.textSecondary};
+  :hover {
+    cursor: pointer;
+    text-decoration: none;
+  }
+`
+
+const USDText = styled.div`
+  font-size: 16px;
+  font-weight: 500;
+  color: ${({ theme }) => theme.textSecondary};
+  margin-top: 8px;
+`
+
+const AuthenticatedHeader = () => {
+  const { account, chainId, connector } = useWeb3React()
+  const [isCopied, setCopied] = useCopyClipboard()
+  const copy = useCallback(() => {
+    setCopied(account || '')
+  }, [account, setCopied])
+  const dispatch = useAppDispatch()
+  const balanceString = useCurrencyBalanceString(account ?? '')
+  const {
+    nativeCurrency: { symbol: nativeCurrencySymbol },
+    explorer,
+  } = getChainInfoOrDefault(chainId ? chainId : SupportedChainId.MAINNET)
+
+  const { address: parsedAddress } = useENS(account)
+  const unclaimedAmount: CurrencyAmount<Token> | undefined = useUserUnclaimedAmount(parsedAddress)
+  const connectionType = getConnection(connector).type
+  const nativeCurrency = useNativeCurrency()
+  const nativeCurrencyPrice = useStablecoinPrice(nativeCurrency ?? undefined) || 0
+  const openClaimModal = useToggleModal(ApplicationModal.ADDRESS_CLAIM)
+
+  const amountUSD = useMemo(() => {
+    const price = parseFloat(nativeCurrencyPrice.toFixed(5))
+    const balance = parseFloat(balanceString || '0')
+    return price * balance
+  }, [balanceString, nativeCurrencyPrice])
+
+  return (
+    <>
+      <div style={{ marginBottom: 12 }}>
+        <div style={{ display: 'inline-block', marginTop: 4 }}>
+          <div style={{ display: 'flex' }}>
+            <StatusIcon connectionType={connectionType} size={24} />
+            <Text fontSize={16} fontWeight={600} marginTop="2.5px">
+              {account && shortenAddress(account, 2, 4)}
+            </Text>
+          </div>
+        </div>
+        <IconContainer>
+          <IconButton onClick={copy} Icon={Copy} text={isCopied ? <Trans>Copied!</Trans> : <Trans>Copy</Trans>} />
+          <IconButton href={`${explorer}address/${account}`} Icon={ExternalLink} text={<Trans>Explore</Trans>} />
+          <IconButton
+            onClick={() => {
```

Comment:
can you do `useCallback` for this? again i know maybe it was copied but should be helpful for future readers